docs/walkthrough/editing-a-package.md

```diff
@@ -107,9 +120,11 @@ If the physical key and the logical key are the same, you may omit the second ar
 
 
 ```python
-# assuming data.csv is in the current directory
-p = quilt3.Package()
-p.set("data.csv")
+import quilt3
+from os import chdir
+# assuming data.csv is in that directory
+q = quilt3.Package()
+q.set("data.csv")
```

Comment:
As I understand this works only because `data.csv` was created in the previous code block. Shouldn't we test each codeblock in some isolated environment? Otherwise codeblock could interfere and became fragile to changes (i.e. modifying one codeblock could break others which seem totally unrelated).

Reply:
Fair. Reverting.